src/components/Pages/resetPassword.tsx

```diff
@@ -0,0 +1,66 @@
+import '../../CSS/login.css'
+
+import Button from 'react-bootstrap/Button';
+import Form from 'react-bootstrap/Form'
+import { PasswordReset } from '../../auth';
+import { useHistory } from 'react-router-dom';
+import { useState } from 'react';
+
+
+export const ResetPassword = () => {
+    const [email, setEmail] = useState('');
+    const [isSuccess, setIsSuccess] = useState(false);
+    const [loading, setLoading] = useState(false);
+    const [error, setError] = useState('');
+    const history = useHistory();
+
+    const handleSubmit = async(event : React.FormEvent<HTMLFormElement>) => {
+        event.preventDefault();
+        setIsSuccess(false);
+        setError('');
+        setLoading(true);
+        const result = await PasswordReset(email);
+        
+        if(result.success){
+            setLoading(false);
+            setIsSuccess(true);
+        }
+
```

Comment:
Remove the extra line
Please do self review before creating PR or adding new changes